src/selfplay/tournament.cc

```diff
@@ -98,8 +100,6 @@ void SelfPlayTournament::PopulateOptions(OptionsParser* options) {
   defaults->Set<bool>(SearchParams::kNoiseId.GetId(), true);
   defaults->Set<float>(SearchParams::kFpuReductionId.GetId(), 0.0f);
   defaults->Set<std::string>(SearchParams::kHistoryFillId.GetId(), "no");
-  defaults->Set<std::string>(NetworkFactory::kBackendId.GetId(),
```

Comment:
Why remove the multiplexing default?  Its more important than ever... ?

Reply:
Ahh I see why, theoretically its not needed if we have enough per thread parallelism.  But I think it is still likely to be useful.

Reply:
Returned multiplexing back.